R/utilities-logging.R

```diff
@@ -0,0 +1,56 @@
+#' @title logError
+#' @description Save error messages into a log error file
+#' @param message message to save in log file
+#' @param file log file name.
+#' Default is `log-error.txt`
+#' @param printConsole logical to print error on console

```

Comment:
The goal of this parameter is to have a way of printing the error or information messages in the console while it is running on the user R session.
If FALSE, it is silent and only printed within the log.
I was thinking that such parameter could be useful for logDebug since you do not really want to have every intermediate printing on the console.
If you think it is not needed, I can remove it. Otherwise, I can define instead a silent mode  parameter which should be clearer.